renku/ui/cli/update.py

```diff
@@ -190,6 +191,11 @@ def update(update_all, dry_run, paths, provider, config, ignore_deleted, skip_me
 
     communicator = ClickCallback()
 
+    if not paths and not update_all and not dry_run:
+        message = f"{WARNING}Updating all outputs could trigger expensive computations. Are you sure?"
+        if not communicator.confirm(message, default=True):
```

Comment:
why not use `abort=True` ?

Reply:
@mohammad-alisafaee You marked the above as resolved but I don't see a corresponding change

Reply:
Sorry, forgot to push the latest version 🤦